webviewloader/module.go

```diff
@@ -10,15 +10,58 @@ import (
 )
 
 var (
-	nativeModule = windows.NewLazyDLL("WebView2Loader")
-	nativeCreate = nativeModule.NewProc("CreateCoreWebView2EnvironmentWithOptions")
+	nativeModule                 = windows.NewLazyDLL("WebView2Loader")
+	nativeCreate                 = nativeModule.NewProc("CreateCoreWebView2EnvironmentWithOptions")
+	nativeCompareBrowserVersions = nativeModule.NewProc("CompareBrowserVersions")
 
-	memOnce   sync.Once
-	memModule winloader.Module
-	memCreate winloader.Proc
-	memErr    error
+	memOnce                   sync.Once
+	memModule                 winloader.Module
+	memCreate                 winloader.Proc
+	memCompareBrowserVersions winloader.Proc
+	memErr                    error
 )
 
+// CompareBrowserVersions will compare the 2 given versions and return:
+//  -1 = v1 < v2
+//   0 = v1 == v2
+//   1 = v1 > v2
+func CompareBrowserVersions(v1 string, v2 string) (int, error) {
+	nativeErr := nativeModule.Load()
+	if nativeErr == nil {
+		nativeErr = nativeCompareBrowserVersions.Find()
+	}
+
+	_v1, err := windows.UTF16PtrFromString(v1)
+	if err != nil {
+		return 0, err
+	}
+	_v2, err := windows.UTF16PtrFromString(v2)
+	if err != nil {
+		return 0, err
+	}
+
+	var result int
+	if nativeErr != nil {
+		loadFromMemory()
+		if memErr != nil {
+			return 0, fmt.Errorf("Unable to load WebView2Loader.dll from disk: %v -- or from memory: %w", nativeErr, memErr)
```

Comment:
Can you move this error value into the `loadFromMemory` function? Both uses of `loadFromMemory` construct it, it may as well just be unified I think.